_rules/image-no-text-0va7u6.md

```diff
@@ -12,7 +12,6 @@ accessibility_requirements:
     inapplicable: further testing needed
   wcag20:1.4.9: # Images of Text (No Exception) (AAA)
     forConformance: true
-    secondary: true
```

Comment:
I did this because I couldn't come up with a sensible explanation of why this was a secondary requirement. The only difference between these two criteria is an assumption for this rule. I think failing this rule under 1.4.5 but not under 1.4.9 in a tool that supports AAA is incorrect.

I don't mind opening a separate PR, but I think that would then just be a blocker to this PR. I was just hoping to avoid that. Let me know if that's okay.